src/Services/Air/templates/AIR_FLIGHT_INFORMATION_REQUEST.xml

```diff
@@ -6,12 +6,14 @@
     <soapenv:Body>
         <air:FlightInformationReq TargetBranch="{{TargetBranch}}">
             <com:BillingPointOfSaleInfo OriginApplication="uAPI"/>
+            {{#flightInfoCriteria}}
             <air:FlightInfoCriteria
                 Carrier="{{airline}}"
                 DepartureDate="{{departure}}"
                 FlightNumber="{{flightNumber}}"
-                Key="{{key}}"
+                Key=""
```

Comment:
Empty key works fine, there is a new `key` in response, but we are not returning the `key` at all.